### PR TITLE
support high contrast colors

### PR DESCRIPTION
**Pull request by birkir**
Update docs to support High Contrast colors, outlined in this PR: https://github.com/facebook/react-native/pull/31651

### CODE REVIEW

**Comment by birkir** [comment]
https://github.com/facebook/react-native/commit/4b9d9dda270acd4e0314f40490c699ffd0f6e30e
v65 - https://github.com/react-native-community/releases/issues/235

File: docs/dynamiccolorios.md [hunk 1]
```diff
@@ -8,14 +8,17 @@ import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import con
 The `DynamicColorIOS` function is a platform color type specific to iOS.
 
 ```jsx
-DynamicColorIOS({ light: color, dark: color });
+DynamicColorIOS({
+  light: color,
+  dark: color,
+  highContrastLight: color, // (optional) will fallback to "dark" if not provided
+  highContrastDark: color // (optional) will fallback to "light" if not provided
```
Comment (Simek) [inline docs/dynamiccolorios.md:15]:
```suggestion
  highContrastLight: color, // (optional) will fallback to "light" if not provided
  highContrastDark: color // (optional) will fallback to "dark" if not provided
```

**Review by Simek — APPROVED** [review]
As you mentioned the feature has landed in `0.65` branch, so this PR is ready to merge to `next` version now. 🙂 

Thank you for posting update to the docs, and working on high contrast color support in the first place. Please fix the comments in the first code block, besides that changes LGTM, great work! 👍

**Comment by birkir** [comment]
Accepted, Thanks  @Simek

**Pull request merged**